### PR TITLE
refactor: delete root_path_dir after run case

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: src/test/util.h [hunk 1]
```diff
@@ -39,11 +41,34 @@ namespace openmldb {
...
-
 inline std::string GenRand() {
     return std::to_string(rand() % 10000000 + 1);  // NOLINT
 }
 
+class TempPath {
+ public:
+    TempPath() {
+        std::filesystem::path tmp_path = std::filesystem::temp_directory_path();
+        base_path_ = absl::StrCat(tmp_path.string(), "/openmldb_test", ::openmldb::test::GenRand());
```
Comment (aceforeverd) [inline src/test/util.h]: filesystem::path supports append with `operator/`

File: src/test/util.h [hunk 2]
```diff
@@ -39,11 +41,34 @@ namespace openmldb {
...
+    }
+    std::string GetTempPath() {
+        return absl::StrCat(base_path_, "/", ::openmldb::test::GenRand());
+    }
+    std::string GetTempPath(const std::string& prefix) {
+        return absl::StrCat(base_path_, "/", prefix, ::openmldb::test::GenRand());
+    }
+    std::string CreateTempPath(const std::string& prefix) {
+        std::string path = GetTempPath(prefix);
+        std::filesystem::create_directory(path);
```
Comment (aceforeverd) [inline src/test/util.h]: check ret value

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: src/test/util.h (hunk 2 above)
Comment (dl239) [reply src/test/util.h]: no need to check, program will core dump if create failed(not catch exception).

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**